// magic-script-components/src/components.js
import React from '../../react/index.js';

function Button (props) {
    // return (<button {...props} />);
    return React.createElement('button', props);
}

function Model (props) {
    // return (<model {...props} />);
    return React.createElement('model', props);
}

function Slider (props) {
    // return (<slider {...props} />);
    return React.createElement('slider', props);
}

function Text (props) {
    // return (<text {...props} />);
    return React.createElement('text', props);
}

function View (props) {
    // return (<view {...props} />);
    return React.createElement('view', props);
}

function Toggle (props) {
    return React.createElement('toggle', props);
}

function TextEdit (props) {
    // return (<textEdit {...props} />);
    return React.createElement('textEdit', props);
}

function Image (props) {
    // return (<image {...props} />);
    return React.createElement('image', props);
}

function ScrollBar (props) {
    // return (<scrollBar {...props} />);
    return React.createElement('scrollBar', props);
}

function ScrollView (props) {
    // return (<scrollView {...props} />);
    return React.createElement('scrollView', props);
}

function Dialog (props) {
    // return (<dialog {...props} />);
    return React.createElement('dialog', props);
}

function CircleConfirmation (props) {
    // return (<circleConfirmation {...props} />);
    return React.createElement('circleConfirmation', props);
}

export { Button, CircleConfirmation, Dialog, Image, Model, ScrollBar, ScrollView, Slider, Text, TextEdit, Toggle, View };
